waspc/src/Wasp/Generator/WebAppGenerator.hs

```diff
@@ -236,9 +231,6 @@ genIndexJs spec =
       )
   where
     maybeSetupJsFunction = AS.App.Client.setupFn =<< AS.App.client (snd $ getApp spec)
-    maybeSetupJsFnImportDetails = getJsImportDetailsForExtFnImport extClientCodeDirInWebAppSrcDirP <$> maybeSetupJsFunction
-    (maybeSetupJsFnImportIdentifier, maybeSetupJsFnImportStmt) =
+    maybeSetupJsFnImportDetails = getJsImportStmtAndIdentifier [reldirP|./|] <$> maybeSetupJsFunction
```

Comment:
The current `getJsImportStmtAndIdentifier` implementation is based around using relative paths and `Extimport` data. My intention was to avoid constructing the relative path to the `ext-src` every time from scratch. If you are using the `getJsImportStmtAndIdentifier` from `WebAppGenerator`, it will know where to import the JS file from.

Again, smilar comment: https://github.com/wasp-lang/wasp/pull/956/files#r1103689737